biogeochem/FatesAllometryMod.F90

```diff
@@ -2634,6 +2966,10 @@ subroutine VegAreaLayer(tree_lai,tree_sai,tree_height,iv,nv,pft,snow_depth, &
     
     tree_vai = tree_lai + tree_sai
 
+    ! Ratio between crown depth and plant height
+    call CrownDepth(tree_height,pft,crown_depth)
+    frac_crown_depth = crown_depth / tree_height
```

Comment:
@rgknox Could this be causing the bit-for-bit mismatch? In the past we would apply the crown depth directly, whereas now we calculate the crown depth and divide by the tree height to get the fraction.

Reply:
Its the same math, so I would only expect round-off level differences here

Reply:
I've tested this out and I can confirm that this isn't causing diffs.